yt_dlp/extractor/afl.py

```diff
@@ -0,0 +1,161 @@
+        'md5': '',
+        'info_dict': {
+            'id': '6dbb9b23-7f00-49d4-b44e-aec2017651dc',
+            'title': 'Walk a Mile in Their Shoes',
+        },
+        'playlist_mincount': 3,
+    }, {
+        'url': 'https://www.afl.com.au/podcasts/afl-daily',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.carltonfc.com.au/podcasts/summer-sessions',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+        element = get_element_by_class('omny-embed', webpage)
+        podcast_url = traverse_obj(extract_attributes(element), ('src', {url_or_none}))
+        return self.url_result(podcast_url, OmnyFMShowIE)
+
+
+class AFCVideoIE(InfoExtractor):
+    IE_NAME = 'afc:video'
+    _VALID_URL = r'https?://(?:www\.)?(?:afc|carltonfc)\.com.au/video/(?P<id>\d+)'
+    _TESTS = [{
+        'url': 'https://www.afc.com.au/video/1657583/girls-academies-be-a-pro?videoId=1657583&modal=true&type=video&publishFrom=1726548621001',
+        'md5': '6b52c149ae6566abe4cfc2d24978983d',
+        'info_dict': {
+            'id': '6362050135112',
+            'ext': 'mp4',
+            'description': 'md5:35897062f9a02043ece73a410bda595c',
+            'upload_date': '20240917',
+            'duration': 103.92,
+            'tags': 'count:0',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'title': 'AFLW Jones Radiology Injury Update: R4',
+            'uploader_id': '6057984922001',
+            'timestamp': 1726558062,
+        },
+    }, {
+        'url': 'https://www.carltonfc.com.au/video/1657596/cripps-on-taking-carlton-to-the-next-level?videoId=1657596&modal=true&type=video&publishFrom=1726555500001',
+        'md5': 'fb5d909329871aa6d182e520d1627846',
+        'info_dict': {
+            'id': '6362089476112',
+            'ext': 'mp4',
+            'description': 'md5:823db447fd9aed2033548e39283d3c0f',
+            'upload_date': '20240918',
+            'duration': 75.72,
+            'tags': 'count:0',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'title': 'The Rundown | Impact of fans',
+            'uploader_id': '6057984922001',
+            'timestamp': 1726631322,
+        },
+    }, {
+        'url': 'https://www.afc.com.au/video/1586280/se10ep16-the-crows-show?videoId=1586280&modal=true&type=video&publishFrom=1719639000001&tagNames=crowsshowepisode',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.carltonfc.com.au/video/1658173/the-rundown-impact-of-fans?videoId=1658173&modal=true&type=video&publishFrom=1726630922001',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+        video_id = self._search_regex(r'"mediaId"\s*:\s*"(\d+)"', webpage, 'video-id')
```

Comment:
There are multiple `mediaId` properties on the webpage. You need to find the one that is playing.

Reply:
This is a blunder on my end by not cross verifying the data properly. Thanks for checking. I'll make sure this doesnt happen again. I've fixed it.